base/src/protocol_state_proof/mod.rs

```diff
@@ -142,3 +142,11 @@ pub struct ProofOpenings {
     pub proof: OpeningProof,
     pub evals: (ProofEvaluations, ProofEvaluations),
 }
+
+pub trait HexConvertable {
```

Comment:
This feels like a strange place for this trait to live. Maybe we can group all the Base58, Base64, Hex conversion traits in the same place

Reply:
Fixed